src/markdown-helpers.ts

```diff
@@ -640,6 +642,13 @@ export const safelyJoinTokens = (tokens: Token[], options: JoinTokenOptions = {}
         break;
       case 'paragraph_open':
       case 'blockquote_close':
+      case 'html_block':
+        break;
+      case 'html_inline':
+        // Replace <br> elements with a newline
+        if (tokenToCheck.content.match(/<br\s*\/?>/)) {
```

Comment:
Do we need the `$` and `^` anchors here? Worst case we get a bunch of newlines but not sure how this works 😅

Reply:
In my testing on https://markdown-it.github.io/ (need to toggle the `html` option), `html_inline` is always going to have a token that starts with `<` and ends with `>`, so I don't think we need anchors.

Reply:
Oh yeah, I was just thinking about some truly garbage HTML
<img width="1051" alt="image" src="https://github.com/electron/docs-parser/assets/6634592/79355931-5725-488b-b4d4-855b8d5d5db8">
Or some less garbage HTML that's just nested
<img width="1247" alt="image" src="https://github.com/electron/docs-parser/assets/6634592/0a34788e-7e6f-4432-9302-e0ab2d4fc567">

Reply:
Those are both `html_block` so they'll get chomped entirely by this change, which is I think what we want?